index.js

```diff
@@ -127,6 +128,11 @@ function pairdown() {
 }
 
 function initPortfolio() {
+  if (process.env.CHE_PROJECTS_ROOT) {
```

Comment:
Ultimately it may not matter if we are planning on abstracting away this logic into a Function but I would recommend being consistent with requiring the environment module and using 

```js
if (env.codenvyUser) {...}
```

Same goes for the other files where you destructure the module export rather than doing the above approach